### PR TITLE
transaction-builder: drop dependency on compiled-stdlib

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: storage/inspector/src/main.rs [hunk 1]
```diff
@@ -41,6 +41,14 @@ enum Command {
     ListAccounts,
 }
 
+/// Returns a user friendly mnemonic for the transaction type if the transaction is
+/// for a known, white listed, transaction.
+fn get_transaction_name(code: &[u8]) -> String {
+    LegacyStdlibScript::try_from(code).map_or("<unknown transaction>".to_string(), |name| {
```
Comment (tzakian) [inline storage/inspector/src/main.rs:47]: Could you possibly switch this to use this new function here instead? https://github.com/diem/diem/blob/master/language/diem-framework/compiled/src/shim/tmp_new_transaction_scripts.rs#L108

**Review by tzakian — APPROVED** [review]
LGTM. Just one small comment

File: storage/inspector/src/main.rs (hunk 1 above)
Comment (bmwill) [reply storage/inspector/src/main.rs]: will do, thanks!

**Pull request merged**